reactioncommerce/reaction-next-starterkit
# refactor: static assets

Resolves #354
Impact: **minor**
Type: **refactor|docs|chore**

## Issue
Unclear and inconsistent handling of static assets.

## Solution
  * Move all static assets into `src/static` dir
  * Updated asset path in configs/env
  * Added docs around static assets

## Breaking changes
locales.json & favicon files are now in `src/static` dir.

## Testing
1. Make sure favicon/placeholder/locales are still loading:
    * When page loads you should always have favicons now and the network path is from "localhost/static/favicons/..."
    * View product that doesn't have an image, placeholder image should render in the grid as well as PDP.
    * Add item to cart and go checkout, during the shipping address checkout step verify the `country` and `region` option are available. 
2. Read over the new static assets docs in `./docs/`.

## Notes
I nested assets in "type" directories (favicons -> static/favicons/...), now I'm thinking it might be best to just keep everything at the root `/static/` dir.
